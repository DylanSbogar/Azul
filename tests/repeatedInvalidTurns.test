RYLYRLRLLLULYYLULYUURYBYYLRUYBLUYULBRUUUUBURRBRRYBYBBUBYRRRLBRULBRYUYRBUULBYYLLBLRLYRUUBRBUYBYLBBLBR
A
B
turn ee wwww foefwinf
turn 11 U 34
turn 5 R 4
turn 0 F 1
turn 4 y 0
