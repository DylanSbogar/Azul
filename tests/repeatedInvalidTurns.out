Factories:
0: F
1: R Y L Y 
2: R L R L 
3: L L U L 
4: Y Y L U 
5: L Y U U

Score for Player A: 0
Mosaic for Player A: 
1         . || . . . . . 
2       . . || . . . . . 
3     . . . || . . . . . 
4   . . . . || . . . . . 
5 . . . . . || . . . . . 
broken: 

Score for Player B: 0
Mosaic for Player B: 
1         . || . . . . . 
2       . . || . . . . . 
3     . . . || . . . . . 
4   . . . . || . . . . . 
5 . . . . . || . . . . . 
broken: 